core/evancore.vue.libs/src/components/permissions-editor/permissions-editor.vue

```diff
@@ -34,7 +34,7 @@
               :permissions="containersPermissions[address].permissions"
               :contract-id="address"
               :update-permissions="updateContractPermissions"
-              :i18n-scope="i18nScope"
+              :i18n-scope="containersPermissions[address].i18nScope || address"
```

Comment:
Doesn't this break older component properties elsewhere?

Reply:
Yes of course, but it is only used by the our current profile and twin dapp. So i just updated the two and adjusted them to use the correct logic :)